src/components/app.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import BooksList from './bookslist';
+import BooksForm from './book-form';
+
+const App = (props) => {
+  const { store } = props;
+  return (
+    <div>
+      <BooksList store={store} />
+      <BooksForm store={store} />
```

Comment:
we don't need to use store here to give to the component access to the data store. please remove store.